### PR TITLE
Rename packages to `org.apache.polaris`

### PR DESCRIPTION
**Pull request by snazy**

### CODE REVIEW

**Review by jbonofre — COMMENTED** [review]
Generally ok, but as we change the header, and also renaming packages, I would request a review from @takidau and/or @RussellSpitzer 

Thanks !

File: regtests/Dockerfile [hunk 1]
```diff
@@ -44,4 +49,39 @@ RUN ./setup.sh
 
 COPY --chown=spark . /home/spark/regtests
 
+# TODO /home/spark/regtests might not be writable in all situations? Seeing those locally :shrug:
```
Comment (takidau) [inline regtests/Dockerfile:52]: Did you mean to leave this in here? Seems better to not make comment additions that don't have anything to do with the package rename in such a giant PR?

**Review by takidau — APPROVED** [review]
One comment, PTAL. But overall LGTM diffed against #173. Thank you!

File: regtests/Dockerfile (hunk 1 above)
Comment (snazy) [reply regtests/Dockerfile:52]: Yea - let me recheck that one.

Comment (snazy) [reply regtests/Dockerfile]: Weird.
I run into a file-permission issue when I run the regtests locally via "*podman* compose" - and it *was* the same error in CI. Looks like something has changed in the meantime and CI is happy now w/o this change. :shrug:

Comment (snazy) [reply regtests/Dockerfile]: Removed this change. Let me tackle that in a different PR - definitely cleaner.

**Review by jbonofre — APPROVED** [review]

**Pull request merged**